src/visualizations/config/adapters/dhis_highcharts/exporting.js

```diff
@@ -0,0 +1,25 @@
+import { VIS_TYPE_SINGLE_VALUE } from '../../../../modules/visTypes.js'
+import { renderSingleValueSVG } from './customSVGOptions/singleValue/renderer/renderSingleValueSVG.js'
+
+export default function getExporting(visType) {
+    const exporting = {
+        // disable exporting context menu
+        enabled: false,
+    }
+    switch (visType) {
+        case VIS_TYPE_SINGLE_VALUE:
+            return {
+                ...exporting,
+                chartOptions: {
```

Comment:
Isn't this returned already in `chart.js`?
Here `getExporting` for SV is returning other things than the config for exporting...

Reply:
I'll add a comment